### PR TITLE
add en doc for vehicle attr

### PR DESCRIPTION
**Pull request by zoooo0820**
add english document for vehicle attribute recognition model.

### CODE REVIEW

**Review by nemonameless — DISMISSED** [review]

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+English | [简体中文](ppvehicle_attribute.md)
+
+# Attribute Recognition Module of PP-Vehicle
+
+Vehicle attribute recognition is widely used in smart cities, smart transportation and other scenarios. In PP-Vehicle, a vehicle attribute recognition module is integrated, which can identify vehicle color and model.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md:5]: PP-Vehicle integrates a vehicle attribute recognition module to identify vehicle color and models.

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+| Task | Algorithm | Precision | Inference Speed | Download |
+|-----------|------|-----------|----------|---------------------|
+| Vehicle Detection/Tracking | PP-YOLOE | - | - | [Inference and Deployment Model](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_ppvehicle.zip) |
+| Vehicle Attribute Recognition | PPLCNet | 90.81 | 2.36 ms | [Inference and Deployment Model](https://bj.bcebos.com/v1/paddledet/models/pipeline/vehicle_attribute_model.zip) |
+
+
+Note:
+1. The prediction speed of the attribute model is based on the test of Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, the MKLDNN acceleration strategy is enabled, and the number of threads is 10.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: prediction和inference都可以的情况下，和上文的表头统一用inference可能会比较好？
base on也不太合适
The inference speed of the attribute model is obtained from the test on Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, with the MKLDNN acceleration strategy enabled, and 10 threads.

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+2. About the model PPLCNet, Please refer to [PP-LCNet Series](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.4/docs/en/models/PP-LCNet_en.md).
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: 这里和原文是不是有点不一样呀
如果需要原文的翻译：
For introductions, please refer to PP-LCNe
Related paper is available on PP-LCNet paper

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+3. The training and test phase of vehicle attribute recognition model are both based on [VeRi dataset](https://www.v7labs.com/open-datasets/veri-dataset).
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: 这里的base on和上文的base on情况一样
可参考点1的处理方式

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+
+- The published pretrained model support the recognition of 10 colors and 9 models, same with VeRi dataset. The detail are:
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: The provided pre-trained model supports 10 colors and 9 models, which is the same with VeRi dataset. The details are as follows:

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+- "bus"
+- "truck"
+- "estate"
+```
+
+## Instructions
+
+### Description of Configuration
+
+Parameters related to vehicle attribute recognition in the [config file](../../config/infer_cfg_ppvehicle.yml) are as follow:
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: as follows

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+  batch_size: 8     # The size of the inference batch
+  color_threshold: 0.5  # Threshold of color. Confidence is required to reach this threshold to determine the specific attribute, otherwise it will be 'Unknown‘.
+  type_threshold: 0.5   # Threshold of vehicle model. Confidence is required to reach this threshold to determine the specific attribute, otherwise it will be 'Unknown‘.
+  enable: False         # Whether to enable this function
+```
+
+### How to Use
+1. Download models `Vehicle Detection/Tracking` and `Vehicle Attribute Recognition` from the links in `Model Zoo` and unzip them to ```./output_inference```. The models are automatically downloaded by default. If you download them manually, you need to modify the `model_dir` as the model storage path.
+
+2. Set the "enable: True" of `VEHICLE_ATTR` in infer_cfg_ppvehicle.yml.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md:64]: to use this function

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+3. For image input, please run these commands. (Description of more parameters, please refer [QUICK_STARTED - Parameter_Description](./PPVehicle_QUICK_STARTED.md).
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: please refer to

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+# For folder contains one or multiple videos
+python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_ppvehicle.yml \
+                                                   --video_dir=test_videos/ \
+                                                   --device=gpu
+```
+
+5. There are two ways to modify the model path:
+
+    - Method 1：Convfigure different model paths in `./deploy/pipeline/config/infer_cfg_ppvehicle.yml`. For vehicle attribute recognition, the path should be modified under the `VEHICLE_ATTR` field.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: 这里中文没有看很懂 是车辆属性识别模型需要修改attr里的路径吗
如果是，翻译可能需要再调整

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+The result is shown as follow:
+
+<div width="1000" align="center">
+  <img src="../images/vehicle_attribute.gif"/>
+</div>
+
+
+### Introduction to the Solution
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: introduction一般用于最开头，也已经用于最开头了
这里适合用summary 
或者features 因为内容是总结了下ppv的特性/优点

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+The vehicle attribute recognition model is from ultra-lightweight image classification scheme (PULC, Practical Ultra Lightweight image Classification), which provided by [PaddleClas](https://github.com/PaddlePaddle/PaddleClas). For details on data preparation, training, and testing of the model, please refer to [PULC Recognition Model of Vehicle Attribute](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.4/docs/en/PULC/PULC_vehicle_attribute_en.md).
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: The vehicle attribute recognition model adopts PULC, Practical Ultra Lightweight image Classification from PaddleClas. 

中文里用括号解释的英文，英文翻译版本可以直接用，不要括号了吧
以及这里原文 车辆属性模型，是正确的还是漏了识别两个字呀

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+
+The vehicle attribute recognition model adopts the lightweight and high-precision PPLCNet. And on the basis of this model, the following optimization scheme is further used:
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: And on top of PPLCNet, our model optimized via:

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+
+- By using the SSLD pre-trained model, the accuracy can be improved by about 0.5 percentage points without changing the inference speed.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: 三个可以统一格式，英文里数据成果喜欢放前面
improved about 0.5 percentage points accuracy by using the SSLD pre-trained model without changing the inference speed
improved 0.52 percentage points accuracy further by integrating EDA data augmentation strategy
类似这种～

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md (hunk 9 above)
Comment (zoooo0820) [reply deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: 是这个意思，重新修改了一版描述，可以再看下呢

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md (hunk 11 above)
Comment (zoooo0820) [reply deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: done，中文文档补充了这个词

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+VEHICLE_ATTR:
+  model_dir: output_inference/vehicle_attribute_infer/ # Path of the model
+  batch_size: 8     # The size of the inference batch
+  color_threshold: 0.5  # Threshold of color. Confidence is required to reach this threshold to determine the specific attribute, otherwise it will be 'Unknown‘.
+  type_threshold: 0.5   # Threshold of vehicle model. Confidence is required to reach this threshold to determine the specific attribute, otherwise it will be 'Unknown‘.
+  enable: False         # Whether to enable this function
+```
+
+### How to Use This function
+1. Download models `Vehicle Detection/Tracking` and `Vehicle Attribute Recognition` from the links in `Model Zoo` and unzip them to ```./output_inference```. The models are automatically downloaded by default. If you download them manually, you need to modify the `model_dir` as the model storage path.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: 是我前面没说清楚哈哈哈
If you download them manually, you need to modify the `model_dir` as the model storage path to use this function
标题里的how to use是ok的

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+
+# For folder contains one or multiple videos
+python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_ppvehicle.yml \
+                                                   --video_dir=test_videos/ \
+                                                   --device=gpu
+```
+
+5. There are two ways to modify the model path:
+
+    - Method 1：Set paths of each model in `./deploy/pipeline/config/infer_cfg_ppvehicle.yml`. For vehicle attribute recognition, the path should be modified under the `VEHICLE_ATTR` field.
```
Comment (Huyushan001) [inline deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md:96]: 这个是ok哒 意思很清楚了

File: deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md (hunk 14 above)
Comment (zoooo0820) [reply deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md]: Done, thanks

**Review by jerrywgz — APPROVED** [review]

**Review by zhiboniu — APPROVED** [review]
